tests/test_tension.py

```diff
@@ -0,0 +1,99 @@
+from anesthetic.examples.perfect_ns import correlated_gaussian
+import numpy as np
+from scipy.linalg import inv, solve, slogdet
+from anesthetic.tension import tension_stats
+
+
+def test_tension_stats_compatiable_gaussian():
+    d = 3
+    V = 1.
```

Comment:
If we really need V to be a float, then this should probably be written as 1.0. However, I don't see any reason 1 wouldn't be fine